tests/dummy/app/styles/app.scss

```diff
@@ -18,7 +17,6 @@
     white-space: -o-pre-wrap;
   }
   code {
-    font-family: Monaco, Consolas, "Andale Mono", "DejaVu Sans Mono", monospace;
```

Comment:
I don't think there is UX defined for code samples, but they better be monospace fonts if you don't wanna drive people nuts